cedar-policy-core/src/entities.rs

```diff
@@ -433,6 +463,129 @@ mod json_parsing_tests {
     use super::*;
     use crate::extensions::Extensions;
 
+    #[test]
+    fn adds_extends_tc() {
```

Comment:
would be great to have another test involving `EnforceAlreadyComputed` and ensure that it (a) works when TC is in fact respected, and (b) fails properly when TC is not respected